config/settings/audit.yml

```diff
@@ -0,0 +1,9 @@
+# The url for this app, also used by `dfe_sign_in`
+base_url: https://audit.register-trainee-teachers.education.gov.uk
```

Comment:
is this going to be a custom domain? or this needs changing to a cloudapps url?

Reply:
they wanted to be customdomain.....